MIGRATION.md

```diff
@@ -0,0 +1,222 @@
+# Migration Guide for Azure Data Explorer (Kusto) Go SDK
+
+Welcome to the migration guide for the Azure Data Explorer (Kusto) Go SDK. This guide is designed to assist you in migrating your application from using the beta version of the SDK (`github.com/Azure/azure-kusto-go`) to the new, stable release, version `1.0.0`.
+
+The release of version `1.0.0` introduces breaking changes, which include changes in dependencies, package arrangement, querying methods, and management commands. Following these steps carefully will ensure a smooth transition to the new version.
+
+## 1. Changes in Dependencies and go.mod File
+
+To migrate from the old SDK to the new one, you'll need to update your `go.mod` file's dependencies.
+
+**Old SDK:**
```

Comment:
why needed ? can just put the new ..

Reply:
Very much, this being a migration guide.